Dockerfiles/BuildTest/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Usage: FROM [image name]
-FROM busybox
+FROM busybox:1.34.1
```

Comment:
I do wonder if we need a script that can check for updated docker images if we're going to hard-code versions. This version seems to include the latest CVE fixes in that package though.

Reply:
As I mentioned in https://github.com/kata-containers/tests/pull/4194#issuecomment-979378569 we can ignore (in-line) the rules [DL3006](https://github.com/hadolint/hadolint/wiki/DL3006) and [DL3007](https://github.com/hadolint/hadolint/wiki/DL3007).

Actually `busybox` a good example of image that we don't expect breakage because of a version bumps, so being entitled to have those rules ignored. What do you think @jodh-intel ?

Reply:
Maybe I should delete that dockerfile instead -> https://github.com/kata-containers/tests/issues/4222